packages/shared/components/Sidebar.svelte

```diff
@@ -153,10 +156,18 @@
                     />
                 </button>
                 <button
-                    class="w-8 h-8 flex items-center justify-center rounded-full bg-{profileColor}-500 leading-100"
+                    class="w-8 h-8 relative flex items-center justify-center rounded-full bg-{profileColor}-500 leading-100"
                     on:click={() => (showProfile = true)}
                 >
                     <span class="text-12 text-center text-white uppercase">{profileInitial}</span>
+                    {#if !isBackupSafe || !$versionDetails.upToDate}
+                        <span class="absolute -top-2 -left-2 flex justify-center items-center h-3 w-3">
+                            <span
+                                class="animate-ping absolute inline-flex h-full w-full rounded-full bg-red-500 opacity-75"
+                            />
+                            <span class="relative inline-flex rounded-full h-2 w-2 bg-red-500" />
+                        </span>
```

Comment:
Lets make this a reusable component, and also use it in the SidebarTab component where we have this code already so we are not repeating ourselves. Could have more than one variant for colour 'warning', 'notification'.